### ISSUE TITLE
separate some of the accessibility-related items from the more general items

### ISSUE THREAD

**Issue by karlgroves** [karlgroves/diagnostic.css#6]
For instance, under deprecated attributes is a selector that checks for the 'target' attribute.  This is an accessibility concern that extends beyond the use of deprecated markup and should be called out separately
